### PR TITLE
[UI] userSelect css bug 

### LINKED ISSUE #1060: (Double) clicking on items causes distracting highlight
## Description

(Double) clicking on items causes distracting highlight.

## Reproduce

1. Open git panel
2. Double click on a file listed as stage/changed or untracked
3. See highlight show up in addition to the blue background indicating that it is active

Screenshot from 2021-12-10 12-47-27

## Expected behavior

No selection is shown on items which respond to click events in a programmatic fashion.

Adding CSS rule: `user-select: none` should solve the issue.

## Context

- Python package version: 0.34.0
- Extension version: v0.34.0

### PR DESCRIPTION
**Pull request by himanshu007-creator**
## Description ❓

This PR resolves #1060

Adds userSelect style to gileitem and branchmenu.

### CODE REVIEW

File: src/style/BranchMenu.ts [hunk 1]
```diff
@@ -115,7 +115,8 @@ export const newBranchButtonClass = style({
 
 export const listItemClass = style(
   {
-    padding: '4px 11px!important'
+    padding: '4px 11px!important',
+    userSelect: null,
```
Comment (krassowski) [inline src/style/BranchMenu.ts:119]:
```suggestion
    userSelect: null
```

File: src/style/BranchMenu.ts [hunk 2]
```diff
@@ -115,7 +115,8 @@ export const newBranchButtonClass = style({
 
 export const listItemClass = style(
   {
-    padding: '4px 11px!important'
+    padding: '4px 11px!important',
+    userSelect: null
```
Comment (fcollonval) [inline src/style/BranchMenu.ts:119]:
```suggestion
    userSelect: 'none'
```

File: src/style/FileItemStyle.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { showButtonOnHover } from './ActionButtonStyle';
 
 export const fileStyle = style(
   {
+    userSelect: null,
```
Comment (fcollonval) [inline src/style/FileItemStyle.ts:6]:
```suggestion
    userSelect: 'none',
```

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @himanshu007-creator 

I'm sorry the correct value is `'none'` not `null`. Would you mind correcting it?

Could you also add that style in the following block for the files list in the history tab:

https://github.com/fcollonval/jupyterlab-git/blob/2343024abe16f56700a2c4a6d7b99dc366c60a11/src/style/SinglePastCommitInfo.ts#L38

File: src/style/BranchMenu.ts (hunk 2 above)
Comment (himanshu007-creator) [reply src/style/BranchMenu.ts:119]: ill do it

**Review by fcollonval — APPROVED** [review]
Thanks a lot @himanshu007-creator and sorry for the late answer

**Pull request merged**